### PR TITLE
Release 0.2.0

### PR DESCRIPTION
**Pull request by luongvo**
https://github.com/luongvo/flutter-survey/milestone/2

## Features

- #11 
- #6 
- #7

## Chores
- #45 
- #25 
- #30

## Bugs
- N/A

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -15,34 +15,48 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
-version: 1.0.0+1
+version: 0.2.0+1
```
Comment (manh-t) [inline pubspec.yaml:18]: oh, so the first version is `1.0.0`?? 😅 btw, you have to increase the build number as well
```suggestion
version: 0.2.0+2
```

Comment (luongvo) [reply pubspec.yaml:18]: @manh-t the build number will be automatically increased by build runner
- https://github.com/luongvo/flutter-survey/pull/54/files#diff-d51300f9a04aa847194d5d92e6d258e82e93953bfc60ff8aacc97dae0cd24a51R38
- https://github.com/luongvo/flutter-survey/pull/54/files#diff-d51300f9a04aa847194d5d92e6d258e82e93953bfc60ff8aacc97dae0cd24a51R75

Comment (manh-t) [reply pubspec.yaml:18]: ah I got it ⚡

**Review by manh-t — APPROVED** [review]

**Comment by luongvo** [comment]
CI has been re-running successfully here https://github.com/luongvo/flutter-survey/actions/runs/1240590045

**Comment by sleepylee** [comment]
should we rerun the failed check? 🤔

**Comment by luongvo** [comment]
> should we rerun the failed check? 🤔

@sleepylee already done above ☝️ 🤓

**Review by sleepylee — APPROVED** [review]
ok I trust you pikachu

**Pull request merged**